src/natcap/invest/urban_nature_access.py

```diff
@@ -316,6 +600,186 @@ def _convert_density_to_population(density):
     shutil.rmtree(tmp_working_dir, ignore_errors=True)
 
 
+def instantaneous_decay_kernel_raster(expected_distance, kernel_filepath,
+        normalize=False):
+    """Create a raster-based, discontinuous decay kernel.
```

Comment:
Is this the dichotomy decay kernel? If so, would it be better to name it as such over 'instantaneous'?

Reply:
Yeah, I was thinking about that too.  I kind of prefer "instantaneous" decay (or "discontinuous", I'd be happy with that too), but the design doc uses dichotomous, so this probably should too.  I've updated this in 5ae39a8f

Reply:
Yea, happy to go with which ever you think. I was just trying to match up the kernel functions to the kernel key, name dictionary.